SocialCareCaseViewerApi/V1/Gateways/CaseStatusGateway.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using SocialCareCaseViewerApi.V1.Helpers;
+using SocialCareCaseViewerApi.V1.Infrastructure;
+using CaseStatus = SocialCareCaseViewerApi.V1.Domain.CaseStatus;
+using CaseStatusField = SocialCareCaseViewerApi.V1.Domain.CaseStatusField;
+using CaseStatusFieldSelectedOption = SocialCareCaseViewerApi.V1.Domain.CaseStatusFieldSelectedOption;
+
+namespace SocialCareCaseViewerApi.V1.Gateways
+{
+    public class CaseStatusGateway : ICaseStatusGateway
+    {
+        private readonly DatabaseContext _databaseContext;
+        private readonly IProcessDataGateway _processDataGateway;
+        private readonly ISystemTime _systemTime;
+
+        public CaseStatusGateway(
+            DatabaseContext databaseContext,
+            IProcessDataGateway processDataGateway,
+            ISystemTime systemTime
+        )
+        {
+            _databaseContext = databaseContext;
+            _processDataGateway = processDataGateway;
+            _systemTime = systemTime;
```

Comment:
Can be dropped from constructor/not passed in anywhere CaseStatusGateway is used.